recipe/recipe.yaml

```diff
@@ -201,11 +208,12 @@ outputs:
             - expecttest
             - scipy
             - pytest-mock
+            - pytest-socket
         script:
           - if: not aarch64
-            then: pytest --verbose -k "not (${{ tests_to_skip }})" --durations=50 test/
+            then: pytest --disable-socket --verbose -k "not (${{ tests_to_skip }})" --durations=50 test/
           - if: aarch64 and (build_platform == target_platform)
-            then: pytest -k "not (${{ tests_to_skip }})" --durations=50 test/
+            then: pytest --disable-socket -k "not (${{ tests_to_skip }})" --durations=50 test/
           - if: aarch64 and (build_platform != target_platform)
             then: true
```

Comment:
What's the reason for separate invocations here? Just the `--verbose`?
Why not
```yaml
        script:
          - if: not aarch64 or (build_platform == target_platform)
            then: pytest --disable-socket --verbose -k "not (${{ tests_to_skip }})" --durations=50 test/
          # nothing else
```

Reply:
Honestly, I have no clue why. I mean — it was like that.

"Nothing else" is not an option since `if`/`then` is handled by preprocessor, and then `aarch64 and (build_platform != target_platform)` yields an invalid empty `script:`.

That said, I'm not sure if we really need the condition in the first place. rattler-build may have improved the support for testing via emulator, so I'll retry that later today.

Reply:
Let's clean this up with the next PR.

Reply:
To be honest, either I'm doing something wrong or the `build_platform != target_platform` condition doesn't seem to work at all. I'm investigating right now.

Reply:
Okay, it's *the other* (i.e. the `python:` group) tests failing, and they're failing both with rattler-build and with the old conda-build recipe (tested as of 477803ab1ff52d0eefe9405105f9ce53a13470cc).